src/test/K6/src/tests/platform/storage/messageboxinstances.js

```diff
@@ -63,13 +64,29 @@ export default function(data) {
         "language": "nb",
         "appId": appOwner + "/" + appName,
         "instanceOwner.partyId": partyId
-    }
+    };
     res = sbl.searchSblInstances(runtimeToken, filters);
     success = check(res, {
         "Search instances by app id status is 200:": (r) => r.status === 200
     });
     addErrorCount(success);
 
+    //Test to get instances based on filter parameters: created and lastChanged from storage: SBL and validate the response
```

Comment:
Is there anyway to have a check on the response time as well? and not just statuscode. ?

Reply:
yes, it is possible to test the response time. I think that should be handled in a performance test script. This script will only be used for functional regression test.